print_job_queue/models.py

```diff
@@ -0,0 +1,80 @@
+from enum import Enum, unique
+
+from django.conf import settings
+from django.db import models
+
+
+@unique
+class PrintJobState(Enum):
+    # The job has been created and is ready for pickup.
+    PENDING = 0
+
+    # The job is being processed by a worker.
+    PROCESSING = 1
```

Comment:
Super nit: grammatically this should be `BEING_PROCESSED`. Alternatively can we have `IN_PROGRESS` instead? I think this is a more well-known term for a job state.

Reply:
Done.